javav2/usecases/creating_fsa_app/src/main/java/com/example/fsa/services/ExtractTextService.java

```diff
@@ -0,0 +1,53 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.fsa.services;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.textract.model.BlockType;
+import software.amazon.awssdk.services.textract.model.Document;
+import software.amazon.awssdk.services.textract.model.DetectDocumentTextRequest;
+import software.amazon.awssdk.services.textract.model.DetectDocumentTextResponse;
+import software.amazon.awssdk.services.textract.model.Block;
+import software.amazon.awssdk.services.textract.TextractClient;
+import software.amazon.awssdk.services.textract.model.S3Object;
+
+public class ExtractTextService {
+
+    public String getCardText(String bucketName, String obName) {
+        Region region = Region.US_EAST_1;
+        TextractClient textractClient = TextractClient.builder()
+            .region(region)
+            .build();
+
+        S3Object s3Object = S3Object.builder()
+            .bucket(bucketName)
+            .name(obName)
+            .build();
+
+        // Create a Document object and reference the s3Object instance.
+        Document myDoc = Document.builder()
+            .s3Object(s3Object)
+            .build();
+
+        DetectDocumentTextRequest detectDocumentTextRequest = DetectDocumentTextRequest.builder()
+            .document(myDoc)
+            .build();
+
+        // Use StringBuilder to build the complete text.
```

Comment:
Is StringBuilder the most relevant piece here?

Reply:
That seems to be a good way to build the string.